model/project_ref.go

```diff
@@ -1243,7 +1243,7 @@ func GetTasksWithOptions(projectName string, taskName string, opts GetProjectTas
 	pipeline = append(pipeline, bson.M{"$sort": bson.M{task.RevisionOrderNumberKey: -1}})
 
 	res := []task.Task{}
-	if _, err = db.AggregateWithMaxTime(task.Collection, pipeline, &res, tasksByProjectQueryMaxTime); err != nil {
+	if err = db.AggregateWithMaxTime(task.Collection, pipeline, &res, tasksByProjectQueryMaxTime); err != nil {
```

Comment:
Didn't know this was a thing. 😎
  I wonder if we should really use this everywhere. We sometimes have allot of db queries that take longer than the request timeout.

Reply:
Yeah this is a nice little func! We can cut a tech-debt ticket to do that work.